KUBERNETES.md

```diff
@@ -0,0 +1,231 @@
+## Running: Kubernetes
+
+This project uses [Lightbend's Platform Tooling](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/) to
+simplify deployment to [Kubernetes](https://kubernetes.io/). Follow the steps below to install Online Auction in your
+own local Kubernetes environment.
+
+## Setup
+
+You'll need to ensure that the following software is installed:
+
+* [Docker](https://www.docker.com/)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl)
+* [Minikube](https://github.com/kubernetes/minikube) v0.25.0 or later (Verify with `minikube version`)
+* [Helm](https://github.com/kubernetes/helm)
+* [reactive-cli](https://developer.lightbend.com/docs/reactive-platform-tooling/latest/cli-installation.html#install-the-cli) 0.9.0 or later (Verify with `rp version`)
+
+## Deploy
+
+Once you've setup your environment, it's time to deploy the application. There are two ways you can do this:
+
+1) Developer Workflow, targeted at developers. This allows you to deploy all of your services with a single SBT task.
+
+2) Operations Workflow, targeted at operations. This matches more closely how you would deploy a Lagom system in production.
+
+### Developer Workflow
+
+> Note that this is an alternative to the Operations workflow documented below.
+
+The following workflow is well suited for development purposes. It will build Docker images for each subproject, generate Kubernetes
+YAML, and deploy them into your Minikube using `kubectl`. It also installs the [Reactive Sandbox](https://github.com/lightbend/reactive-sandbox)
+in your Minikube, providing the required dependencies for Kafka, Cassandra, and Elasticsearch.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+```bash
+minikube addons enable ingress
+minikube start --memory 6000
+
+sbt "deploy minikube"
+```
+
+> Open the URL this command prints in the browser
+
+```bash
+echo "http://$(minikube ip)"
+```
+
+You should now be able to use the application with your browser. Please note that since Minikube uses a self-signed
+certificate for TLS, it will result in a browser warning.
+
+### Operations Workflow
+
+> Note that this is an alternative to the Development workflow documented above.
+
+This workflow matches more closely how an application would be deployed in a production environment. In this method,
+the images are built by SBT. The [reactive-cli](https://github.com/lightbend/reactive-cli) is then
+used to generate YAML for deployment to the Kubernetes cluster. The steps below focus on Minikube, but could be used on
+a real Kubernetes cluster given access to a Docker registry and configured `kubectl`.
+
+> Want a fresh Minikube? Run `minikube delete` before the steps below.
+
+##### Start Minikube
```

Comment:
I would start numbering the rest of the headings from this point on for the Operations Workflow. i.e. #### 1) Start Minikube,  And, in the intro, explain that it is a N-step process.

#### 2) Set up Docker engine ....

Reply:
:+1: